src/main/java/org/vaadin/example/bookstore/ui/inventory/InventoryView.java

```diff
@@ -94,39 +95,73 @@ public void showError(String msg) {
         Notification.show(msg);
     }
 
-    public void showSaveNotification(String msg) {
+    /**
+     * shows a message in a notification.
+     *
+     * @param msg
+     */
+    public void showNotification(String msg) {
         Notification.show(msg);
     }
 
+    /**
+     * Enables/Disables the new product button.
+     *
+     * @param enabled
+     */
     public void setNewProductEnabled(boolean enabled) {
         newProduct.setEnabled(enabled);
     }
 
+    /**
+     * Clears the fields in the product form.
```

Comment:
Is this correct, doesn't it just deselect the row in Grid?

Reply:
Done.